### PR TITLE
Proofs about Permutation Cycles

### PR DESCRIPTION
**Pull request by tirix**
This includes several proofs which were pending on my side, but no major new result.

One interesting result might be `~symgcntz`, which states that any set of permutations with disjoint orbits commute, and the corollary for permutation cycles, `~tocyccntz`.

A few theorems were moved from other mathboxes to the main part.

### CODE REVIEW

**Review by savask — APPROVED** [review]
Nice work! Always good seeing more group theory in set.mm.

File: set.mm [hunk 1]
```diff
@@ -457631,6 +457714,36 @@ Class abstractions (a.k.a. class builders)
...
+    $( Rewrite an empty intersection with a pair.  (Contributed by Thierry
+       Arnoux, 20-Nov-2023.) $)
+    inpr0 $p |- ( ( A i^i { B , C } ) = (/) <-> ( -. B e. A /\ -. C e. A ) ) $=
+      ( vx cv wne wa wral cpr cin c0 wceq wcel wn r19.26 wi wal wb 3bitr4i nelb
+      cvv nelpr elv imbi2i albii disj1 df-ral anbi12i ) DEZBFZUICFZGZDAHZUJDAHZ
+      UKDAHZGABCIZJKLZBAMNZCAMNZGUJUKDAOUIAMZUIUPMNZPZDQUTULPZDQUQUMVBVCDVAULUT
+      VAULRDUIBCUAUBUCUDUEDAUPUFULDAUGSURUNUSUODBATDCATUHS $.
+  $}
+
+  $( The class ` A ` is not in ` ( C \ { A , B } ) ` .  (Contributed by Thierry
```
Comment (avekens) [inline set.mm:457735]: This is just the statement slightly rewritten. Maybe "An element of a pair is not an element of a difference with this pair."

File: set.mm [hunk 2]
```diff
@@ -524408,6 +524956,15 @@ A Fne if ( S = (/) , { X } , U. S ) ) $=
   mofal $p |- E* x F. $=
     ( wfal wex wmo nexfal exmo mtpor ) BACBADAEBAFG $.
 
+  ${
+    rmonot.1 $e |- ( x e. A -> -. ph ) $.
+    $( "At most one" restricted existential quantifier for a statement never
+       true.  (Contributed by Thierry Arnoux, 27-Nov-2023.) $)
```
Comment (avekens) [inline set.mm:524962]: "... which is never true."

File: set.mm (hunk 1 above)
Comment (tirix) [reply set.mm:457735]: Yes! I was not very inspired.
I'll write "_The first element of a pair is not an element of a difference with this pair._" and similarly for the second such theorem.

**Review by avekens — APPROVED** [review]

**Pull request merged**